runtime-modules/proposals/discussion/src/tests/mock.rs

```diff
@@ -9,7 +9,7 @@ use sp_core::H256;
 use sp_runtime::{
```

Comment:
Shouldn't we change the frame_system::AccountId type to u128 after introducing `module_id` based accounts?